src/cljs/replumb/ast.cljs

```diff
@@ -39,68 +40,75 @@
    (get-in state [:cljs.analyzer/namespaces ns :defs])))
 
 (defn ns-defs
-  "Given compiler state and namespace symbol, returns its AST :defs
-  content (beware, it can be a lot of data)"
+  "Given compiler state and namespace symbol, returns its AST's `:defs`
+  key content.
+
+  Beware, there can be a lot of data."
   [state ns]
   {:pre [(symbol? ns)]}
   (get-in state [:cljs.analyzer/namespaces ns :defs]))
 
 (defn namespace
   "Given compiler state and namespace symbol, returns its whole AST
-  content (beware, it can be a lot of data)."
+  content.
+
+  Beware, there can be a lot of data."
   [state ns]
   {:pre [(symbol? ns)]}
   (get-in state [:cljs.analyzer/namespaces ns]))
 
 (defn requires
-  "Return required symbols given compiler state and a namespace: a
-  map of {ns ns}.
+  "Return required symbols given compiler state and namespace: a map of
```

Comment:
I opted to remove `s` in the first sentence only, I see it somewhere, they use imperative (?) tense